src/components/Schedule.jsx

```diff
@@ -285,8 +274,13 @@ const Schedule = ({ courseOrder, jsonSched, aliases, showInstructorNames }) => {
     <>
       <img
         src={dataURL}
-        className={classes.Media}
-        style={{ visibility: dataURL ? "visible" : "hidden" }}
+        style={{
+          height: "auto",
+          width: "auto",
+          maxHeight: "90%",
+          maxWidth: "90%",
+          visibility: dataURL ? "visible" : "hidden",
+        }}
```

Comment:
Not a fan of using inline like this, I'll see if I can make styled components work with this.